### PR TITLE
Polishing README.md

### PR DESCRIPTION
**Pull request by concretevitamin**
Tested (run the relevant ones):

- [ ] Code formatting: `bash format.sh`
- [ ] Any manual or new tests for this PR (please specify below)
- [ ] All smoke tests: `pytest tests/test_smoke.py` 
- [ ] Relevant individual smoke tests: `pytest tests/test_smoke.py::test_fill_in_the_name` 
- [ ] Backward compatibility tests: `bash tests/backward_comaptibility_tests.sh`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,28 +31,32 @@
 - [June, 2023] [**Two new clouds supported**](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html): Samsung SCP and Oracle OCI!
 - [April, 2023] **[**SkyPilot YAMLs released**](./llm/vicuna/) for finetuning & serving the Vicuna model with a single command**!
 - [March, 2023] **[Vicuna LLM chatbot](https://lmsys.org/blog/2023-03-30-vicuna/) trained** [**using SkyPilot**](./llm/vicuna/) **for $300 on spot instances!** 
-- [March, 2023] *Serve* your own LLaMA LLM chatbot (not finetuned) on any cloud: [**example**](./llm/llama-chatbots/), [**repo**](https://github.com/skypilot-org/sky-llama)
+- [March, 2023] Serve your own LLaMA LLM chatbot (not finetuned) on any cloud: [**example**](./llm/llama-chatbots/), [**repo**](https://github.com/skypilot-org/sky-llama)
 ----
 
-SkyPilot is a framework for easily and cost effectively running ML workloads[^1] on any cloud. 
+SkyPilot is a framework for running LLMs, AI, and Batch jobs on any cloud, offering maximum cost savings, highest GPU availability, and managed execution.
```
Comment (romilbhardwaj) [inline README.md:37]:
```suggestion
SkyPilot is a framework for running LLMs, AI, and batch jobs on any cloud, offering maximum cost savings, highest GPU availability, and managed execution.
```

Seems like batch job is not a proper noun - [Google results](https://www.google.com/search?q=Batch+jobs&oq=Batch+jobs&aqs=chrome..69i57j69i59l2j69i61l2.1031j0j1&sourceid=chrome&ie=UTF-8#ip=1)

File: README.md [hunk 2]
```diff
@@ -31,28 +31,32 @@
...
 
-SkyPilot abstracts away the cloud infra burden:
-- Launch jobs & clusters on any cloud (AWS, Azure, GCP, Lambda Cloud, IBM, Samsung, OCI)
-- Find scarce resources across zones/regions/clouds
-- Queue jobs & use cloud object stores
+SkyPilot **abstracts away cloud infra burdens**:
+- Launch jobs & clusters on any cloud 
+- Easy scale-out: queue and run many jobs, automatically managed
+- Easy access to cloud buckets
```
Comment (romilbhardwaj) [inline README.md:42]: Suggestion, feel free to ignore: Consider adding (S3, GCS and R2) -  [Google trends popularity of S3 vs 'Cloud buckets'](https://trends.google.com/trends/explore?date=all&geo=US&q=Cloud%20bucket,S3&hl=en-GB)

**Review by romilbhardwaj — APPROVED** [review]
Thanks @concretevitamin!

File: README.md (hunk 2 above)
Comment (concretevitamin) [reply README.md]: Added. Also changed to "object stores" which is more popular: https://trends.google.com/trends/explore?date=all&geo=US&q=Cloud%20bucket,object%20store&hl=en-GB

**Review by romilbhardwaj — APPROVED** [review]

**Pull request merged**